nautobot/extras/api/serializers.py

```diff
@@ -954,10 +1012,12 @@ class Meta:
             "destination_label",
             "destination_hidden",
             "destination_filter",
+            "created",
+            "last_updated",
```

Comment:
Did you find that you needed to add these? `ValidatedModelSerializer` should be automatically injecting these two fields...

Reply:
So, these lines are on RelationshipSerializer. I did not add these, but it could have been added during merge resolution.